projects/plugins/jetpack/modules/videopress/utility-functions.php

```diff
@@ -731,7 +731,11 @@ function videopress_get_attachment_url( $post_id ) {
 			return null;
 		}
 	} else {
-		$return = $meta['videopress']['file_url_base']['https'] . $meta['videopress']['files']['hd']['mp4'];
+		$return = $meta['videopress']['file_url_base']['https'] . (
+			$meta['videopress']['files']['hd']['hls']
```

Comment:
Should we check if `$meta['videopress']['files']['hd']['hls']` is set before we check for it here, just like we do for `$meta['videopress']['files']['hd']['mp4']` above?

Reply:
Sure, I can change the condition here to be more explicit with an `isset()`.